netbeans.apache.org/src/content/about/history.asciidoc

```diff
@@ -0,0 +1,166 @@
+a business was born. The original business plan was to develop network-enabled
+JavaBeans components. Jaroslav Tulach, who designed the IDE's basic
+architecture, came up with the name NetBeans to describe what the components
+would do. The IDE would be the way to deliver them. When the specification for
+Enterprise Java Beans came out, it made more sense to work with the standard
+for such components than to compete with it--but the name stuck.
+
+In the spring of 1999, NetBeans DeveloperX2 was released, supporting Swing. The
+performance improvements that came in JDK 1.3, released in the fall of 1999,
+made NetBeans a viable choice for development tools. By the summer of 1999, the
+team was hard at work re-architecting DeveloperX2 into the more modular
+NetBeans that forms the basis of the software today.  
+
+== More Than An IDE... and Open Sourced
+
+Along the way, an interesting thing happened. People began building
+applications using the NetBeans IDE's platform, together with their own
+plugins, often creating applications that were not development tools at all. In
+fact, this turned out to have quite a market. Later, in 2000 and 2001, a lot of
+work went into stripping out pieces that made the assumption that an
+application built on NetBeans was an IDE, so that the platform would be a
+generic desktop application suitable to any purpose. This work turned out to be
+healthy for the codebase of the IDE as well, encouraging a clean API design and
+a separation of concerns.
+
+Something else was afoot in the summer of 1999. Sun Microsystems wanted better
+Java development tools, and had become interested in NetBeans. It was a dream
+come true for the NetBeans team. NetBeans would become the flagship tool set of
+the maker of Java itself! By the Fall, with the next generation of NetBeans
+Developer in beta, a deal was struck. Sun Microsystems had also acquired
+another tools company, Forté, at the same time, and decided to rename NetBeans
+to Forté for Java. The name NetBeans was dropped--for a while.
+
+During the acqusition, the young developers who had been involved in
+open-source projects for most of their programming careers, mentioned the idea
+of open-sourcing NetBeans. Fast forward to less than six months later, the
+decision was made that NetBeans would be open sourced. While Sun had
+contributed considerable amounts of code to open source projects over the
+years, this was Sun's first sponsored open source project, one in which Sun
+would be paying for the site and handling the infrastructure. The very first
+decision made was that it sounded logical to call the new site: NetBeans.org.
+In June 2000, the initial netbeans.org web site was launched. The years that
+followed focused on continual enhancements from release to release, as
+described in the section below.
+
+When Oracle acquired Sun in 2010, NetBeans became part of Oracle. Oracle
+actively seeked for new developers to work on the NetBeans team and sees
```

Comment:
FYI: The JDeveloper story might be interesting: http://wiki.apidesign.org/wiki/JDeveloper